README.md

```diff
@@ -1,8 +1,38 @@
 # aws-prototyping-sdk
 
-AWS Prototyping SDK (PDK) is a package which contains useful CDK and Projen constructs.
+## Introduction
 
-## Getting Started
+The AWS Prototyping SDK (PDK) aims to accelerate the development of prototypes on AWS. To achieve this, it provides building blocks for common patterns together with development tools to manage and build your projects. The constructs are based on [AWS CDK](https://github.com/aws/aws-cdk) and implement an expanding number of common application/infrastructure patterns, such as the ability to generate complete client and infrastructure code for a REST API from an OpenAPI specification. The project and build tools the PDK provide allow you manage the configuration of multiple related projects in a language-agnostic way, and to efficiently excute parallel incremental builds.
+
+This compbination of project management, build execution, and CDK constructs reduce the effort involved in prototyping new ideas on AWS, and allow developers to seamlessly grow these ideas beyond the prototype phase without requiring a complete re-implementation of the original solution. In addition to the foundation of project configuration and build dependency management provided by the nx-monorepo project, PDK also provides a number of other projen projects (some of which implement common infrastructure patterns using CDK) and stand-alone CDK constructs, which are described below.
+
+## Core Modules
+
+### [nx-monorepo](./packages/nx-monorepo/README.md)
+
+Modern applications are often implented across multiple packages or libraries, each potentially implemented in a different language. PDK provides a [Projen](https://github.com/projen/projen) project called 'nx-monorepo' to simplify and standardise configuration and build across packages in this context. Together with the project-as-code configuration management provided by Projen, this project utilises [Nx Build](https://nx.dev/) to give devlopers the ability to manage a collection of different projects within a single repository. Nx Build allows for explicit dependency tracking between projects, and implements an efficient build caching mechanism to support fast, parallel incremental builds in a language agnostic manner. By combining Projen and NX Build, 'no-monorepo' reduces the effort involved in maintaining and building related projects implemented in a mix of languages.
```

Comment:
Consider: "Nx Build allows for explicit [and implicit] dependency [management] between polyglot projects, and provides shared caching and parallel task execution for super fast builds across multiple languages with ease."